Update whats-new.mdx (#1744)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

--><!-- END: New Entry Template -->

<!-- BEGIN: What's New Entries -->
## July 19-23, 2021

> - Begun iterating Notifications taxonomy based on consuming of research results

Reviewer: ```suggestion
- Iterated on Notifications taxonomy based on research results
```

> - Continued [ideating on writing guidance for Navigation] (https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20)

Reviewer: ```suggestion
- Continued [ideating on written guidance for Navigation] (https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20)
```

- Audited all of Figma files to ensure correct use of MetricHPEXS font.

> 

Reviewer: ```suggestion
```

> - Audited all of Figma files to make sure we're using the correct MetricHPEXS font

Reviewer: ```suggestion
- Audited all of Figma files to ensure correct use of MetricHPEXS font
```

## July 12-16, 2021

> - Began Notifications Taxonomy Survey

Reviewer: ```suggestion
- Began Notifications Taxonomy Survey
```
```suggestion
- Began Notifications Taxonomy Survey.
```

> - Updated [Header colors on DateInput component in Figma](https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=0%3A1)

Reviewer: ```suggestion
- Updated [Header colors on DateInput component in Figma](https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=0%3A1).
```

> - Removed [Selected state from Menu on Figma file](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1) to align with Design System site

Reviewer: ```suggestion
- Removed [Selected state from Menu on Figma file](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1) to align with Design System site.
```

> - Updated [Data Table with Search Value](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A242) to be updated to match the styling we show in Filtering Figma/on DS site.

Reviewer: ```suggestion
- Updated [DataTable with Search value](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A242) to be updated to match the styling we show in Filtering Figma/on DS site.
```

> - [Updated Select dropdown items on Figma file to use regular text weight rather than form](https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/HPE-Select-Component?node-id=0%3A1)

Reviewer: ```suggestion
- [Updated Select dropdown items on Figma file to use regular text weight rather than form value weight](https://www.figma.com/file/fZJdRcNrvZ1JjdPRYHpFFU/HPE-Select-Component?node-id=0%3A1)
```

> - Updated text styling on [drop list items for TextInput from form/value to text/default](https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component?node-id=0%3A1)

Reviewer: ```suggestion
- [Updated TextInput dropdown items on Figma file to use regular text weight rather than form value weight](https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component?node-id=0%3A1)
```

> - Enhanced the way Menu was built on Figma by making it [one component rather than a button with a list drop.](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1)

Reviewer: ```suggestion
- Enhanced the way Menu is built on Figma by making it [one component rather than a button with a list drop.](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=0%3A1)
```

- Updated Wizard Figma file to match the Design System site. [Changes include: wizard header background color, spacing and text styling](https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizarding?node-id=0%3A1)

## July 5-9, 2021

> - Began ideating guidance and examples for Navigation

Reviewer: ```suggestion
- Began ideation for Navigation guidance and examples
```

> - Added guidance on [how to determine the width of columns on a Table]( https://design-system.hpe.design/components/table#setting-the-width-of-the-table)

Reviewer: ```suggestion
- Added guidance on [how to determine the width of columns on a Table](https://design-system.hpe.design/components/table#setting-the-width-of-the-table).
```

## June 28 - July 2, 2021

> - Added [new ToolBar Button to Figma and it's variants] ( https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1)

Reviewer: ```suggestion
- Added [new Toolbar Button to Figma and it's variants] ( https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1)
```

- Updated [Form Figma file to include correct text styling.](https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=2668%3A3582) TextInput values were changed from text-strong to default text color.

> - Completed r{esearch for Navigation}(https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18)

Reviewer: ```suggestion
- Completed [research for Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18).
```

Reviewer: ```suggestion
- Completed r{esearch for Navigation}(https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18).
```

## June 21-25, 2021

> - Began research explorations for Navigation

Reviewer: ```suggestion
- Began research explorations for Navigation.
```
